src/components/application_manager/include/application_manager/application_impl.h

```diff
@@ -487,9 +483,7 @@ class ApplicationImpl : public virtual Application,
   Timer video_stream_suspend_timer_;
   Timer audio_stream_suspend_timer_;
 
-#ifdef SDL_REMOTE_CONTROL
   std::list<AppExtensionPtr> extensions_;
-#endif  // SDL_REMOTE_CONTROL
 
   /**
    * @brief Defines number per time in seconds limits
```

Comment:
shall the following stuff be moved to vehicle_info plugin?

  bool SubscribeToIVI(uint32_t vehicle_info_type) OVERRIDE;
  bool IsSubscribedToIVI(uint32_t vehicle_info_type) const OVERRIDE;
  bool UnsubscribeFromIVI(uint32_t vehicle_info_type) OVERRIDE;
  DataAccessor<VehicleInfoSubscriptions> SubscribedIVI() const OVERRIDE;

Reply:
@yang1070 all functions regarding subscriptions are already moved to vehicle_info plugin, please look at the latest updates